maquetado/scss/login.scss

```diff
@@ -0,0 +1,66 @@
+.login-container {
+  background-color: $wild-sand;
+  border-top: 8px solid $cerulean;
+  display: flex;
+  flex-flow: column wrap;
+  height: 100%;
+  justify-content: center;
+  max-height: 400px;
+  max-width: 300px;
+  padding: 20px 24px 24px;
+  width: 100%;
+
+  .wolox-logo {
+    height: 70px;
+    max-height: 100%;
+    max-width: 212px;
+    width: 100%;
+  }
+
+  .login-form {
+    border-bottom: 2px solid $grey;
+    display: flex;
+    flex-flow: column wrap;
+    flex-grow: 1;
+    margin-top: 16px;
```

Comment:
We try to use margin-bottom instead of margin-top when we need to separate to components. So you can switch this margin to margin-bottom: 16px and move it to `wolox-logo` class

Reply:
Done.